src/Analyzers/CSharp/CodeFixes/InlineDeclaration/CSharpInlineDeclarationCodeFixProvider.cs

```diff
@@ -70,14 +70,7 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
         await editor.ApplyExpressionLevelSemanticEditsAsync(
             document,
             originalNodes,
-            t =>
-            {
-                using var _ = ArrayBuilder<SyntaxNode>.GetInstance(capacity: 2, out var additionalNodesToTrack);
-                additionalNodesToTrack.Add(t.identifier);
-                additionalNodesToTrack.Add(t.declarator);
-
-                return (t.invocationOrCreation, additionalNodesToTrack.ToImmutable());
-            },
+            static t => (t.invocationOrCreation, ImmutableArray.Create<SyntaxNode>(t.identifier, t.declarator)),
```

Comment:
in some cases, i just literally fixed things up to avoid any builders whatsoever